tools/fastdds/fastdds.py

```diff
@@ -113,7 +111,11 @@ def discovery(self):
 
     def xml(self):
         """Discovery server command handler."""
-        XMLParser(sys.argv[2:])
+        try:
```

Comment:
Why is it done differently from the other CLI verbs? Maybe a comment can clarify this point to future developers looking at this script.

Reply:
In order to let the previous users to keep using the other CLI verbs, and since new CLI verb has this new dependency, this try-catch import has been introduced to advice users to install new dependency if they want to use the new verb.